Makefile

```diff
@@ -2,7 +2,7 @@ all: build
 
 init:           ## Installs tools to $GOPATH/bin (which is expected to be in $PATH).
 	curl https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin
-	go install ./vendor/github.com/jstemmer/go-junit-report
+	go install github.com/jstemmer/go-junit-report
```

Comment:
Now version of this tool will be "random". I think we should pin it, as we do it in platform repos

Reply:
@Dasio ^